lambdasoup.opam

```diff
@@ -28,6 +28,7 @@ pin-depends: [
 
 build: [
   ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest"] {with-test}
```

Comment:
Perhaps `ounit` (above) should be tagged with `with-test` rather than `dev`, given this new command.

Reply:
Indeed; done.